lgtm-generator-from-dropbox.rb

```diff
@@ -88,22 +89,15 @@ def generate_lgtm(file:, text:, color:, size:, gif:, cjk_font:)
 gif = params['gif']
 cjk_font = json_data['cjk_font']
 
-# Select LGTM string's color from inverted color of original image's average color
+# Select LGTM string's color from inverted high contrast color of original image's average color
 if params['auto-color']
   puts 'Auto Color Selecting ...'
-  rgb_color = Color.get_average_color_as_rgb(file: ORIGINAL_IMAGE_NAME)
-  inverted_rgb_color = Color.to_inverted_color_as_rgb(rgb_color)
-
-  # NOTE: RMagick need hex color code
-  inverted_hex_color = Color.rgb_to_hex(
-    inverted_rgb_color[:red],
-    inverted_rgb_color[:green],
-    inverted_rgb_color[:blue]
-  )
+  rgb_color = ColorTone.get_average_color_as_rgb(file: ORIGINAL_IMAGE_NAME)
+  inverted_rgb_color = ColorTone.calc_contrast_color_as_rgb(rgb_color)
```

Comment:
IMO: If `calc_contrast_color_as_rgb` will generate `inverted_rgb_color`, we should name it as like `calc_contrasted_inverted_color_as_rgb` or so. What do you think?

Reply:
It's a great idea! :tada:
I named `calc_inverted_high_contrast_color_as_rgb`.